Components/navBar/navbar.css

```diff
@@ -1,67 +1,70 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap');
 
 .navbar{
-    display: flex;
-    align-items: center;
-    position: sticky;
-    top: 0;
-    width: 100%;
-    height: 80px;
-    box-shadow: 0px 5px 3px 0px #a6a6a6;
```

Comment:
you can globally define box shadow for consistency all over the app. just a suggestion.